app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  respond_to :json
+
+  private
+
+  def respond_with(resource, _opts = {})
+    resource.persisted? ? register_success : register_failed
+  end
+
+  def register_success
+    render json: { message: 'Signed up.' }
+  end
+
+  def register_failed
+    render json: { message: "Signed up failure." }
```

Comment:
Small things, single quote or double quote, let's stick to one and make sure we follow